### PR TITLE
SolarSystem: Use unicode superscripts, multiplication symbol

### PR DESCRIPTION
**Pull request by zachthompson**
## Description of new Instant Answer, or changes
The display is soon to break on our html/lite sites.  

solar-system-broken

This uses unicode superscripts and multiplication symbol instead of html/entities.

solar-system-uni

See if it makes sense.

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [x] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [x] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [x] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [x] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [x] There is no unnecessary files in place (such as editor config files)
- [x] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <fathead_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/solar_system

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/SolarSystem.pm [hunk 1]
```diff
@@ -75,15 +76,17 @@ handle query_lc => sub {
...
     }
     
     # Superscript for scientific notation
-    # Convert x to HTML entity &times;
+    # Convert x to unicode
     if($result =~ m/x\s(10)(\d\d)/) {
         my ($number, $exponent) = ($1, $2);
-        $result =~ s/$number$exponent/$number<sup>$exponent<\/sup>/;
-        $result =~ s/x/&times;/;
+        my $uniexp = $exponent =~ tr/0123456789/⁰ⁱ²³⁴⁵⁶⁷⁸⁹/r;
```
Comment (mintsoft) [inline lib/DDG/Goodie/SolarSystem.pm]: I'd probably put the transliteration here and line 79 into a sub personally to remove the duplication

**Comment by mintsoft** [comment]
@zachthompson do you still want this merging in?

**Comment by mintsoft** [comment]
I think you'll have to rebase, it looks like the tests are running under the wrong ubuntu

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

**Pull request merged**